mmv1/templates/terraform/examples/network_connectivity_regional_endpoint_global_access.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_compute_network" "my_network" {
+  name                    = "<%= ctx[:vars]['rep_network'] %>"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "my_subnetwork" {
+  name          = "<%= ctx[:vars]['rep_subnetwork'] %>"
+  ip_cidr_range = "192.168.0.0/24"
+  region        = "us-central1"
+  network       = google_compute_network.my_network.id
+}
+
+resource "google_network_connectivity_regional_endpoint" "<%= ctx[:primary_resource_id] %>" {
+  name              = "<%= ctx[:vars]['rep_name'] %>"
+  location          = "us-central1"
+  target_google_api = "boqcodelabjaimin-pa.us-central1.p.rep.googleapis.com"
```

Comment:
Does the `target_google_api` need to actually exist? `boqcodelabjaimin-pa.us-central1.p.rep.googleapis.com` is likely not guaranteed to stick around.

Reply:
It's the target_google_api value we use in our prod e2e tests and yeah it has to exist. It's a required field.